src/main/java/org/jitsi/jigasi/AbstractGatewaySession.java

```diff
@@ -236,6 +238,17 @@ void notifyJvbRoomJoined()
         }
     }
 
+    /**
+     *  Method called by {@link JvbConference} to notify session that it has
+     *  detected a recording status change.
+     *
+     * @param mode the recording mode.
+     * @param status the recording status.
+     */
+    void notifyForRecordingStatusChange(
```

Comment:
I was going to suggest that we use `onRecordingStatusChanged` for consistency, but we have 4 methods named with "on" and 5 methods named with "notify"...
How about `notifyRecordingStatusChanged`?